projects/js-packages/publicize-components/src/components/connection-toggle/style.scss

```diff
@@ -7,8 +7,14 @@
 	width: 100%;
 
 	// Unchecked state.
-	&.is-not-checked .jetpack-gutenberg-social-icon {
-		fill: $gray-300;
+	&.is-not-checked {
+		.components-connection-icon__picture img {
+			filter: saturate(0);
+		}
+		.jetpack-gutenberg-social-icon {
+			fill: $gray-300;
+			filter: saturate(0);
```

Comment:
I think this second `filter: saturate(0);` shouldn't be here. Does nothing

Reply:
The whole rule wasn't doing anything as the class name was wrong, but without this filter, the filter on the avatar causes it to be rendered on top of the network icon. We also only render the icon when the avatar isn't available for any reason, so this means the icon can also be toggled.